tests/test_base.py

```diff
@@ -0,0 +1,54 @@
+from api import models
+from django.test import Client
+from django.contrib.auth import login
+from django.test import TestCase, Client
+from rest_framework.test import force_authenticate, APIRequestFactory, APIClient
+from django.urls import reverse
+from .factories import UserFactory, ItemFactory, GroupFactory, CollectionFactory, MeetingFactory
+
+
+class SimpleTest(TestCase):
+    meetings_url = '/collection/1'
+
+    def setUp(self):
+
+        # All users have password "password123" by default
+        # Username defaults to first_initial+last_name (i.e. jdoe)
+        self.owner = UserFactory(username="owner")
+        self.owner.save()
+
+        self.random_user = UserFactory(username="random_user")
+        self.random_user.save()
+
+        self.submitter = UserFactory(username="submitter")
+        self.submitter.save()
+
+        # all foreign keys must be specified, everything else is auto-generated by default, or can be left blank
+        # make a collection with 100 items
+        self.collection = CollectionFactory(created_by=self.owner)
+        self.collection.save()
+
+        items = ItemFactory.build_batch(100, collection=self.collection, created_by=self.submitter)
+        for i in items:
+            i.save()
+
+        # make a meeting with 100 items
+        self.meeting = MeetingFactory(created_by=self.owner)
+        self.meeting.save()
+
+        items = ItemFactory.build_batch(100, collection=self.meeting, created_by=self.submitter)
+        for i in items:
+            i.save()
+
+        self.factory = APIRequestFactory()
+
+    def test_cant_post_collection_unless_logged_in(self):
```

Comment:
I think this test can be split up further, one testing the `201` status code and the other testing the `401` status code